app/app.js

```diff
@@ -1,60 +1,70 @@
-var app = angular.module("myCalculator", []);
+var app = angular.module("myCalculator", ['ngRoute']);
+app.config(['$routeProvider', function($routeProvider){
+	$routeProvider.when('/', {
+		templateUrl : 'templates/home.html',
+		controller : 'CalculatorCtrl'
+	}).when('/newMeal', {
+		templateUrl : 'templates/new-meal.html',
+		controller: 'CalculatorCtrl'
+	}).when('/earnings', {
+		templateUrl: 'templates/my-earnings.html',
+		controller: 'CalculatorCtrl'
+	});
+}]);
+app.controller('CalculatorCtrl', ['$scope', function($scope){
 
-app.controller("CalculatorCtrl", function(){
-
-	this.initCharge = function (){
-		this.subtotalCharge = 0;
-		this.tipCharge = 0;
-		this.totalCharge = 0;	
+	$scope.initCharge = function (){
+		$scope.subtotalCharge = 0;
+		$scope.tipCharge = 0;
+		$scope.totalCharge = 0;	
 	};
 
-	this.initEarnings = function(){
-		this.tipTotal = 0;
-		this.mealCount = 0;
-		this.tipAverage = 0;
+	$scope.initEarnings = function(){
+		$scope.tipTotal = 0;
+		$scope.mealCount = 0;
+		$scope.tipAverage = 0;
 	};
 
 	//Trigered by Cancel button
-	this.cancelValues = function(){
-	this.price = "";
-	this.tax = "";
-	this.tip ="";
+	$scope.cancelValues = function(){
+	$scope.price = "";
+	$scope.tax = "";
+	$scope.tip ="";
  	};
 
  	//Trigered by Reset Button
- 	this.resetAll = function() {
- 		this.errorMsg= "";
- 		this.cancelValues();
- 		this.initCharge();
- 		this.initEarnings();
+ 	$scope.resetAll = function() {
+ 		$scope.errorMsg= "";
+ 		$scope.cancelValues();
+ 		$scope.initCharge();
+ 		$scope.initEarnings();
  	};
 
  	//initial Values of App
- 	this.resetAll();
+ 	$scope.resetAll();
 
  	//Submit Events 
-	this.submitForm = function(){
-		if(this.myForm.$invalid){
-			this.errorMsg = "Please enter valid numeric values";
+	$scope.submitForm = function(){
+		if($scope.myForm.$invalid){
+			$scope.errorMsg = "Please enter valid numeric values";
 		} else {
-			this.errorMsg = "";
-			this.subtotalCharge = this.price * (1 + (this.tax)/100);
-			this.tipCharge = this.subtotalCharge * (this.tip / 100); 
-	 		this.totalCharge = this.subtotalCharge + this.tipCharge;
-			this.mealCount++;
-			this.finalTips();
-			this.averageTip();
+			$scope.errorMsg = "";
+			$scope.subtotalCharge = $scope.price * (1 + ($scope.tax)/100);
+			$scope.tipCharge = $scope.subtotalCharge * ($scope.tip / 100); 
```

Comment:
you should make $scope.tip / 100 and ($scope.tax)/100 have the same formatting (either remove the spaces, or make them consistent